src/app/components/Card/Card.tsx

```diff
@@ -8,12 +8,17 @@ type CardProps = {
     evaluations: string[];
   };
   deleteCard: (name: string) => void;
+  handleClick: () => void;
 };
 
-export default function Card({ pupil, deleteCard }: CardProps): JSX.Element {
+export default function Card({
+  pupil,
+  deleteCard,
+  handleClick,
+}: CardProps): JSX.Element {
   const { name, evaluations } = pupil;
   return (
-    <Container>
+    <Container onClick={handleClick}>
```

Comment:
This is also triggered when `RoundActionButton` is clicked. I recommend using a link as a wrapper for this container instead of the onclick prop